src/foraging_gui/Foraging.py

```diff
@@ -2043,14 +2047,18 @@ def _Save(self,ForceSave=0,SaveAs=0,SaveContinue=0):
         Obj['PhotometryFolder']=self.PhotometryFolder
         Obj['MetadataFolder']=self.MetadataFolder
         
+        # only run once for each session
         if SaveContinue==0:
             # force to start a new session; Logging will stop and users cannot run new behaviors, but can still modify GUI parameters and save them.                 
             self.unsaved_data=False 
             self._NewSession()
+            self.unsaved_data=True
```

Comment:
Why mark `unsaved_data=True`?

Reply:
Just be strict. The data has not been saved at this time.